src/nix/profile.md

```diff
@@ -9,10 +9,12 @@ them to be rolled back easily.
 
 # Default profile
 
-The default profile used by `nix profile` is `$HOME/.nix-profile`,
-which, if it does not exist, is created as a symlink to
-`/nix/var/nix/profiles/default` if Nix is invoked by the
-`root` user, or `/nix/var/nix/profiles/per-user/`*username* otherwise.
+The default profile used by `nix profile` is `~/.nix-profile`, unless
+a setting `use-xdg-base-directories` is set to `true`, in which case
+`$XDG_STATE_HOME/ nix/profile` (or `~/.local/state/nix/profile` if
+`XDG_DATA_HOME` is not set) is used. If that profile does not exist,
```

Comment:
This sentence is getting out of hand. We need some place to document which files are used where, in a structured manner. If we give these files a name which is not their path, we can document them separately, handle the conditionals in that one place, and refer to that part everywhere instead of sprinkling conditional fragments all over the place.

We already have listings of files used by each command, such as for nix-env, nix-channel, and nix profile, and we could give them symbolic names and anchors.